cfme/configure/access_control.py

```diff
@@ -82,19 +144,18 @@ def __exit__(self, *args, **kwargs):
             self._restore_user = None
 
     def create(self):
-        navigate_to(self, 'Add')
-        fill(self.user_form, {'name_txt': self.name,
-                              'userid_txt': self.credential.principal,
-                              'password_txt': self.credential.secret,
-                              'password_verify_txt': self.credential.verify_secret,
-                              'email_txt': self.email,
-                              'user_group_select': getattr(self.group,
-                                                           'description', None)},
-             action=form_buttons.add)
-        flash.assert_success_message('User "{}" was saved'.format(self.name))
+        view = navigate_to(self, 'Add')
+        view.fill({'name_txt': self.name,
+                   'userid_txt': self.credential.principal,
+                   'password_txt': self.credential.secret,
+                   'password_verify_txt': self.credential.verify_secret,
+                   'email_txt': self.email,
+                   'user_group_select': getattr(self.group, 'description', None)})
+        view.add_button.click()
```

Comment:
I'm pretty sure the page will be changed after clicking the button. So I propose to create an appropriate view:
```python
view = self.create_view(RequiredViewHere)
```
Also consider to do this in other parts of your code.